secboot/secboot_sb.go

```diff
@@ -174,6 +206,11 @@ func UnlockVolumeUsingSealedKeyIfEncrypted(disk disks.Disk, name string, sealedE
 		res.UnlockMethod = UnlockedWithSealedKey
 	}
 
+	// FIXME: replace with https://github.com/canonical/secboot/pull/331 when available
```

Comment:
```suggestion
	// keys used to activate the volume were placed in the user keyring, 
	// but with key IDs that match the /dev/disk/by-uuid/.. paths, however
	// if an older snapd is running later on in the host system, it will 
	// attempt to authenticate various FDE operations with keys which
	// formerly used to be stored by secboot at key IDs that match 
	// /dev/disk/by-partuuid/.. device paths. To ensure that new snap-boostrap
	// remains compatible with older snapd, copy the disk unlock and aux keys
	// to key IDs matching the legacy paths.
	// FIXME: replace with https://github.com/canonical/secboot/pull/331 when available
```